doc-content/enterprise-only/bpmn/bpmn-end-events-ref.adoc

```diff
@@ -23,54 +26,52 @@ h|End
 |image:BPMN2/bpmn-end-error.png[]
 
 |<<_compensation_end_event>>
-|
+|image:BPMN2/bpmn-end-compensation.png[]
 
 |<<_escalation_end_event>> 
-|
+|image:BPMN2/bpmn-end-escalation.png[]
 
 |<<_terminate_end_event>>
 |image:BPMN2/bpmn-end-terminate.png[]
 
 
 |===
 
+[[_none_end_event]]
+.None
+
+The none end event specifies that there is no other special behavior associated with the end of the process.
 
 [[_message_end_event]]
 .Message
 
-When a flow enters a Message End Event, the flow finishes and the end event produces a message as defined in its properties.
+When a flow enters a message end event, the flow finishes and the end event produces a message as defined in its properties.
 
 [[_signal_end_event]]
 .Signal
 
-A throwing Signal End Event is used to finish a process or sub-process flow. When the execution flow enters the element, the execution flow finishes and produces a signal identified by its [property]``SignalRef`` property.
+A throwing signal end event is used to finish a process or subprocess flow. When the execution flow enters the element, the execution flow finishes and produces a signal identified by its `SignalRef` property.
 
 
 [[_error_end_event]]
 .Error
 
-The Throwing Error End Event finishes the incoming workflow, that means consumes the incoming token, and produces an error object. Any other running workflows in the process or sub-process remain uninfluenced.
-
-
-* Attributes
-+
-`ErrorRef`
-The reference code of the produced error object.
+The throwing error end event finishes the incoming workflow, that means consumes the incoming token, and produces an error object. Any other running workflows in the process or subprocess remain uninfluenced.
```

Comment:
"workflow, that means consumes ..." is grammatically wonky. I'd say "workflow, which means it consumes ..." but then that's still not great because "which means" is undesirable. But better than the current way, imo. Search "that means". Comes up a couple of times.